hoot-rnd/src/main/cpp/hoot/rnd/conflate/network/DebugNetworkMapCreator.cpp

```diff
@@ -22,7 +22,7 @@
  * This will properly maintain the copyright information. DigitalGlobe
  * copyrights will be updated automatically.
  *
- * @copyright Copyright (C) 2015, 2016 DigitalGlobe (http://www.digitalglobe.com/)
+ * @copyright Copyright (C) 2015, 2016, 2017 DigitalGlobe (http://www.digitalglobe.com/)
```

Comment:
I'm not sure but I can only assume that Network conflation wasn't around in `2015` can be removed.